src/lib/cache/subCache.cpp

```diff
@@ -33,6 +33,7 @@
 
```

Comment:
Sure.
True, I did a pretty thorough test of #2216 and was unable to reproduce it.